maintainer/install_hole.sh

```diff
@@ -32,6 +32,12 @@ set -o errexit -o nounset
 
 SCRIPT=$(basename $0)
 
+<<<<<<< HEAD
```

Comment:
this needs merging correctly, not sure why it caused a conflict

Reply:
I think I rebased already pushed commits while trying out something and hence it caused a conflict. It should be fine now that my local is back to the original state.